Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -333,6 +333,14 @@ def _get_org_id(self, org_name):
             return org_uid
         raise Code42OrgNotFoundError(org_name)
 
+    def get_departing_employee(self, username):
+        user_id = self._get_user_id(username)
+        return self._get_sdk().detectionlists.departing_employee.get(user_id)
+
+    def get_high_risk_employee(self, username):
+        user_id = self._get_user_id(username)
+        return self._get_sdk().detectionlists.high_risk_employee.get(user_id)
```

Comment:
What's this response look like? Depending on, you might want to just return `json` here like the other commands.

Reply:
See Alan's draft PR for an example

Reply:
Changed.